azurelinuxagent/common/cgroupapi.py

```diff
@@ -391,24 +394,38 @@ def remove_extension_cgroups(self, extension_name):
             raise CGroupsException("Failed to remove {0}. Error: {1}".format(unit_filename, ustr(e)))
 
     def get_extension_cgroups(self, extension_name):
-        scope_name = self._get_extension_scope_name(extension_name)
-        cpu_cgroup_path = os.path.join(CGROUPS_FILE_SYSTEM_ROOT, 'cpu', 'system.slice', scope_name)
-        memory_cgroup_path = os.path.join(CGROUPS_FILE_SYSTEM_ROOT, 'memory', 'system.slice', scope_name)
+        slice_name = self._get_extension_cgroup_name(extension_name)
+
+        cgroups = []
+
+        def create_cgroup(controller):
+            cpu_cgroup_path = os.path.join(CGROUPS_FILE_SYSTEM_ROOT, controller, 'system.slice', slice_name)
```

Comment:
Just checking - `.scope` goes in here. Correct?

Reply:
This one actually returns the slice for the extension (which we are not using currently). I'll add ".scope" in start_extension_command